estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,24 @@
+from odoo import api, models, fields
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offer"
+
+    price = fields.Float()
+    status = fields.Selection(copy=False, selection=[('accepted', 'Accepted'), ('refused', 'Refused')])
+    partner_id = fields.Many2one('res.partner', required=True)
+    property_id = fields.Many2one('estate.property', required=True)
+    validity = fields.Integer(default=7)
+    date_deadline = fields.Date(compute='_compute_date_deadline', inverse="_inverse_date_deadline")
+
+    @api.depends("validity", "create_date")
+    def _compute_date_deadline(self):
+        for record in self:
```

Comment:
```suggestion
        for offer in self:
```
record is fine but sometimes with try to give more explicit variable name when iterating over self